### PR TITLE
Build and metadata cleanup and fixes

### PR DESCRIPTION
**Pull request by adamretter**
* Fixes some issues with the build process.
* Adds missing Java dependencies and updates versions for eXist-db 5.3.0.
* Adds and enforces correct license headers.
* Adds missing index.html back into the XAR.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -143,6 +164,67 @@
...
+							<header>LGPL2.1-template.txt</header>
+							<properties>
+								<owner>${project.organization.name}</owner>
+								<organization>The eXist-db Authors</organization>
+								<url>${project.organization.url}</url>
+							</properties>
+							<includes>
+								<include>pom.xml</include>
+								<include>src/test/java/xquery/crypto/CryptoIT.java</include>
+								<!-- include>src/test/java/ro/kuberam/libs/java/crypto/TestUtils.java</include>
```
Comment (line-o) [inline pom.xml:212]: I would remove code instead of commenting it out. It will stay in version control if we ever need to get back to it.

**Review by line-o — COMMENTED** [review]
Good stuff!
One minor suggestion.

**Review by joewiz — APPROVED** [review]
Looks good! And I didn't realize we could generate HTML from Markdown and include the README as index.html in the package. This is going to benefit people who get the package from outside the repo. All good stuff.

File: pom.xml (hunk 1 above)
Comment (joewiz) [reply pom.xml:212]: Sorry, Juri, GitHub didn't show me your comment until after I merged!

**Pull request merged**